README.md

```diff
@@ -7,38 +7,38 @@
 **Kover** - Gradle plugin for Kotlin code coverage tools: [Kover](https://github.com/JetBrains/intellij-coverage)
 and [JaCoCo](https://github.com/jacoco/jacoco).
 
-Minimal supported `Gradle` version: `6.6`.
+Minimum supported version of `Gradle` is `6.6`.
 
 ## Table of contents
+
 - [Features](#features)
 - [Quickstart](#quickstart)
-  - [Apply plugin to a project](#apply-plugin)
-    - [Applying plugins with the plugins DSL](#applying-plugins-with-the-plugins-dsl)
-    - [Legacy Plugin Application: applying plugins with the buildscript block](#legacy-plugin-application-applying-plugins-with-the-buildscript-block)
```

Comment:
It's not that far away from ToC, so probably no point in it taking 3 lines

Reply:
This is OK, because I plan that only the applying of the plugin will remain in the README, and all the rest of the content will be moved to the HTML documentation of the release.